README.md

```diff
@@ -1,7 +1,30 @@
-# SP1 Blobstream
 
-## Overview
+Forked from [sp1-blobstream using commit cbd1ee173a9acf8cda80bf4b6ed093623dd7e0a9, supporter sp1 v3.0.0](https://github.com/succinctlabs/sp1-blobstream/tree/cbd1ee173a9acf8cda80bf4b6ed093623dd7e0a9)
 
-Implementation of zero-knowledge proof circuits for [Blobstream](https://docs.celestia.org/how-to-guides/blobstream), a Celestia's data attestation bridge in SP1.
+## Build & Run
 
-**[Docs](https://succinctlabs.github.io/sp1-blobstream)**
+Circuit Building to Create Elf:
+
+```sh
+cd ./program
+cargo prove build --docker --tag v3.0.0 --elf-name fuelstreamx-elf
+```
+
+Running: 
+
+```sh
+cargo run --release --bin operator
+```
+
+To get the genesis parameters for the smart-contract:
+
+```sh
+cargo run --release --bin genesis
+```
+
+## Tests
+
+```sh
+cd ./scripts
+cargo test
+```
```

Comment:
Didn't manage to build on my machine. But I am assuming that I'm missing something.

Reply:
forgot to include https://docs.succinct.xyz/docs/getting-started/install